clients/go/zpu/cmd/tools/main.go

```diff
@@ -0,0 +1,83 @@
+package main
```

Comment:
Missing license information:

// Copyright 2017 Yahoo Holdings, Inc.
// Licensed under the terms of the Apache version 2.0 license. See LICENSE file for terms.